cogs/commands.py

```diff
@@ -399,6 +399,35 @@ async def result(self, ctx, msg_link: str):
                 await ctx.send(embed=embed)
         except:
             return await ctx.send("That message is not a poll!")
+    
+    @suggestion.command(name="edit", aliases=["change"]
```

Comment:
Please test your code before pushing.

Reply:
Obvious error right here.

Reply:
oops, I have a bunch more edits too I want to make... I'll fix them :+1: